src/main/resources/templates/springboot/rest/component-test-annotation.java.tmpl

```diff
@@ -10,7 +10,7 @@ import java.lang.annotation.Target;
 
 @Target(ElementType.TYPE)
 @Retention(RetentionPolicy.RUNTIME)
-@ActiveProfiles({"local", "componentTest"})
+@ActiveProfiles({"local", "test", "componentTest"})
```

Comment:
why is `test` needed now?

Reply:
to enable core tests

Reply:
specifically, b/c `application-test.properties` contains the datasource url so it will work for both core and component tests